core/common/app.php

```diff
@@ -86,6 +86,7 @@ public function get_name() {
 	 * @access public
 	 */
 	public function register_scripts() {
+		// BC Support.
```

Comment:
- import the code into the common it self.
- remove the dependency of this script.
- if the script is enqueued - put a deprecation notice.